deps/build.jl

```diff
@@ -1,3 +1,12 @@
+version = "master"
```

Comment:
Let's keep this at the latest release? `master` makes it hard to keep things consistent across users as people might have different versions all around. Also it might break without notice.